torch/_inductor/decomposition.py

```diff
@@ -65,6 +65,7 @@
         aten.native_layer_norm,
         aten.nll_loss2d_backward,
         aten._softmax,
+        aten.select_scatter,
```

Comment:
Sorry but none of the `_scatter` ops can be decomposed since we need these ops to be in the graph for the reinplace pass.